src/Microsoft.VisualStudio.Editors/PropPages/PackagePropPage.vb

```diff
@@ -231,7 +233,39 @@ Namespace Microsoft.VisualStudio.Editors.PropertyPages
         ''' </summary>
         Private Sub NeutralLanguageComboBox_DropDown(sender As Object, e As EventArgs) Handles NeutralLanguageComboBox.DropDown
             PopulateNeutralLanguageComboBox(NeutralLanguageComboBox)
-            Common.SetComboBoxDropdownWidth(NeutralLanguageComboBox)
+            SetComboBoxDropdownWidth(NeutralLanguageComboBox)
+        End Sub
+
+        Private Sub LicenseBrowseButton_Click(sender As Object, e As EventArgs) Handles LicenseBrowseButton.Click
+            Dim sInitialDirectory As String = Nothing
+            Dim sFileName As String
+
+            If sInitialDirectory = "" Then
+                sFileName = ""
+                sInitialDirectory = ""
+            Else
+                sFileName = Path.GetFileName(sInitialDirectory)
+                sInitialDirectory = Path.GetDirectoryName(sInitialDirectory)
+            End If
+
+            Dim fileNames As ArrayList = GetFilesViaBrowse(ServiceProvider, Handle, sInitialDirectory, My.Resources.Microsoft_VisualStudio_Editors_Designer.PPG_AddExistingFilesTitle,
+                    CombineDialogFilters(
+                        CreateDialogFilter(My.Resources.Microsoft_VisualStudio_Editors_Designer.RSE_Filter_Text, "txt"),
```

Comment:
What is the value of `RSE_Filter_Text` and `PPG_AddExistingFilesTitle` here?

Reply:
These are for looking up the localized strings in the browse for file menu.
RSE_Filter_Text is "Text Files": 
<img width="148" alt="textfile" src="https://user-images.githubusercontent.com/36282608/48589415-4e677c00-e8f0-11e8-8c89-87e8d473e177.PNG">
PPG_AddExistingFilesTitle is "Add existing file to project":
<img width="125" alt="filetitle" src="https://user-images.githubusercontent.com/36282608/48589437-5fb08880-e8f0-11e8-82d4-a164b774abe9.PNG">